Modules/core/Model/CoreInstall.php

```diff
@@ -642,6 +642,11 @@ public function upgrade_v3_v4() {
                 $eventHandler->customerImport();
                 Configuration::getLogger()->debug('[stats] import clients, done!');
             }
+
+            Configuration::getLogger()->debug("[core_adminmenu] change menu to structure");
+            $sql = "UPDATE core_adminmenu SET name='Structures' WHERE name='Menus'";
```

Comment:
why changing in db ?

Reply:
there are in coreadminemnus some checks about this , only translator should be modified

Reply:
to have "structure" instead of "menu" in admin menu too:

![admin_menu](https://user-images.githubusercontent.com/48911130/144862335-c843ea98-c79a-4aad-ae09-a7c4f728e316.png)

Future superadmins maybe won't know that "structure" was "menu" at first...